README.md

```diff
@@ -25,7 +25,7 @@ Going forward, all images shall be available at `quay.io/oauth2-proxy/oauth2-pro
 
     a. Download [Prebuilt Binary](https://github.com/oauth2-proxy/oauth2-proxy/releases) (current release is `v7.4.0`)
 
-    b. Build with `$ go get github.com/oauth2-proxy/oauth2-proxy/v7` which will put the binary in `$GOROOT/bin`
+    b. Build with `$ go install github.com/oauth2-proxy/oauth2-proxy/v7@latest` which will put the binary in `$GOROOT/bin`
```

Comment:
We have this same line in the documentation site as well, do you think you'd be able to update it there as well?

Reply:
Sure, I just did